sdks/java/core/src/main/java/org/apache/beam/sdk/annotations/Experimental.java

```diff
@@ -76,6 +76,12 @@
     TIMERS,
 
     /** Experimental APIs related to customizing the output time for computed values. */
-    OUTPUT_TIME
+    OUTPUT_TIME,
+
+    /**
+     * <a href="https://s.apache.org/splittable-do-fn">Splittable DoFn</a>.
+     * Do not use: API is unstable and runner support is incomplete.
+     */
+    SPLITTABLE_DO_FN,
```

Comment:
Hmm, not sure what you mean here? Would you like stronger guarantees that users won't be able to apply this transform?